tests/benchmarks/test_encoding.py

```diff
@@ -24,31 +26,25 @@ def decode(data):
         return msgpack.unpackb(data, raw=True)
 
 
-trace_large = gen_trace(nspans=1000)
+# Set a static seed to ensure gen_trace is consistant
+random.seed(13)
```

Comment:
@P403n1x87 is this the right place for this? should we put it in `gen_trace` instead or maybe a `conftest.py` ?

Reply:
If we want to control the seed for all tests then yes we should either put it in `test_encoders` or even better in a `conftest.py`. Perhaps we should also set `PYTHONHASHSEED`, e.g. `PYTHONHASHSEED=0` before running `pytest` to also control the hash seed.